src/js/tabs/osd.js

```diff
@@ -1043,6 +1051,8 @@ OSD.chooseFields = function () {
                     if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
                         OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
                             F.FLIP_ARROW,
+                            F.MAX_ESC_TEMP,
```

Comment:
No, this is the wrong place for these fields again. They are not display, but statistic fields, and need to go into `STATISTIC_FIELDS`.

Also, please test, and if they don't work as expected, fix your changes before pushing them.